docker-compose.yml

```diff
@@ -1,10 +1,24 @@
 version: '2'
 services:
+  mysql:
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_USER: sharetribe
+      MYSQL_PASSWORD: secret
+      MYSQL_DATABASE: sharetribe_production
+      MYSQL_ROOT_PASSWORD: secret
```

Comment:
Should these be stored in `.env`?

Reply:
not necessarily. I was thinking about that, seems like a zero-sum game.